tyk-pro/scripts/bootstrap_k8s.sh

```diff
@@ -1,28 +1,176 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 set -e
 
-apk --no-cache add curl
+apk --no-cache add curl jq
 curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
 install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
 
-NAMESPACE=tyk
+NAMESPACE=$TYK_POD_NAMESPACE
```

Comment:
This `NAMESPACE` variable isn't really needed, `TYK_POD_NAMESPACE` can be used directly.

Reply:
yeah okay!